src/webhook_launcher/app/misc.py

```diff
@@ -46,21 +47,28 @@ def body_callback(self, buf):
         self.contents += buf
 
     def api_call(self, endpoint, call):
-        c = pycurl.Curl()
-        c.setopt(pycurl.SSL_VERIFYPEER, 0)
-        c.setopt(pycurl.SSL_VERIFYHOST, 0)
+        proxies = {}
+        auth = None
         if settings.OUTGOING_PROXY:
-            c.setopt(pycurl.PROXY, settings.OUTGOING_PROXY)
-            c.setopt(pycurl.PROXYPORT, settings.OUTGOING_PROXY_PORT)
-        c.setopt(pycurl.NETRC, 1)
+            proxy = "%s:%s" % (
+                settings.OUTGOING_PROXY,
+                settings.OUTGOING_PROXY_PORT
+            )
+            proxies = {'http': proxy, 'https': proxy}
+        if settings.BB_API_USER:
+            auth = requests.auth.HTTPBasicAuth(
+                settings.BB_API_USER,
+                settings.BB_API_PASSWORD
+            )
         url = str("/%s/%s/%s" % (endpoint, self.slug, call)).replace("//", "/")
         url = self.base + url
-        c.setopt(pycurl.URL, url)
-        c.setopt(c.WRITEFUNCTION, self.body_callback)
-        c.perform()
-        c.close()
+        response = requests.get(
+            url,
+            verify=False,
+            proxies=proxies,
+            auth=auth,
+        )
+        return response.json()
 
     def branches_tags(self):
-        self.api_call('repositories', 'branches-tags')
-        return json.loads(self.contents)
```

Comment:
If self.contents and self.body_callback aren't used anymore, might as well remove them.